debian/changelog

```diff
@@ -1,3 +1,42 @@
+adsys (0.12.0) mantic; urgency=medium
```

Comment:
If you want to merge it before we really release it (as there is still one of two fixes in the pipe), keep the release `UNRELEASED`. Then, you can merge it anytime (squash it to a single commit probably).